src/condor_tools/htcondor_cli/annex.py

```diff
@@ -125,8 +125,8 @@ class Create(Verb):
         "test": {
             "args": ("--test",),
             "help": argparse.SUPPRESS,
-            "default": False,
-            "action": "store_true",
+            "type": int,
+            "default": None,
```

Comment:
--test now requires an int to follow it; have you made sure this doesn't impact any of your existing tests?